relayer/src/cache.rs

```diff
@@ -15,6 +15,10 @@ const CONNECTION_CACHE_TTL: Duration = Duration::from_secs(10 * 60);
 const CLIENT_STATE_CACHE_TTL: Duration = Duration::from_millis(500);
 const LATEST_HEIGHT_CACHE_TTL: Duration = Duration::from_millis(200);
 
+const CHANNEL_CACHE_CAPACITY: u64 = 10_000;
+const CONNECTION_CACHE_CAPACITY: u64 = 128;
```

Comment:
I figured there may be not many connections (you only need one per pair of chains per application?), but the number of channels could be larger. Please suggest more realistic numbers, maybe taking some stats from existing deployments.

Reply:
At the moment, we believe that most channels use their very own connection and that there are probably as many channels as connections, so it would make sense to have the same capacity for both caches.